pom.xml

```diff
@@ -73,6 +73,13 @@
     </modules>
 
     <profiles>
+        <profile>
+            <id>use-ignite-src</id>
```

Comment:
I'd suggest `attach-ignite-src` as it is widely used

Reply:
Could you explain why 'attach'? 
Looks like a random name instead of proper? 
My wish is to rename every 'attach' to `use`, any objections?

Reply:
@ivandasch ^